2016-12-30T07:06:14.385Z - info: 127.0.0.1 - - [30/Dec/2016:07:06:14 +0000] "GET /v2/subcategory/Owl HTTP/1.1" 200 - "-" "PostmanRuntime/3.0.5"

2016-12-30T07:06:25.594Z - error: uncaughtException: Cannot read property 'subCategory' of undefined date=Fri Dec 30 2016 12:36:25 GMT+0530 (IST), pid=12839, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=136036352, heapTotal=104991840, heapUsed=85331088, loadavg=[1.39306640625, 1.52001953125, 1.63134765625], uptime=301810
TypeError: Cannot read property 'subCategory' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/login/server/controllers/login.server.controller.js:247:25
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/utils.js:96:12)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/cursor.js:844:16
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:156:5)
    at setCursorNotified (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:506:3)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:572:16
    at Object.queryCallback [as cb] (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:238:5)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:443:18
    at authenticateStragglers (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:399:16)
    at null.messageHandler (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:421:5)
    at Socket.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/connection.js:294:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:536:20)
2016-12-30T07:08:25.347Z - info: 127.0.0.1 - - [30/Dec/2016:07:08:25 +0000] "GET /v2/subcategory/undifined HTTP/1.1" - - "-" "PostmanRuntime/3.0.5"

2017-01-02T06:38:38.754Z - info: 127.0.0.1 - - [02/Jan/2017:06:38:38 +0000] "POST /v2/confirm-vendor HTTP/1.1" 200 33 "-" "PostmanRuntime/3.0.5"

2017-01-02T06:40:37.735Z - info: 127.0.0.1 - - [02/Jan/2017:06:40:37 +0000] "POST /v2/confirm-vendor HTTP/1.1" 200 33 "-" "PostmanRuntime/3.0.5"

2017-01-02T06:42:45.621Z - info: 127.0.0.1 - - [02/Jan/2017:06:42:45 +0000] "POST /v2/confirm-vendor HTTP/1.1" 200 33 "-" "PostmanRuntime/3.0.5"

2017-01-02T06:43:58.948Z - info: 127.0.0.1 - - [02/Jan/2017:06:43:58 +0000] "POST /v2/confirm-vendor HTTP/1.1" 200 33 "-" "PostmanRuntime/3.0.5"

2017-01-02T07:19:57.620Z - error: uncaughtException: connection timeout date=Mon Jan 02 2017 12:49:57 GMT+0530 (IST), pid=17108, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=39772160, heapTotal=76126304, heapUsed=73660344, loadavg=[2.0849609375, 2.38818359375, 2.40185546875], uptime=561836
Error: connection timeout
    at null.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:168:17)
    at emitTwo (events.js:87:13)
    at emit (events.js:172:7)
    at listener (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/node_modules/mongodb/lib/db.js:1791:14)
    at emitOne (events.js:77:13)
    at emit (events.js:169:7)
    at null.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/node_modules/mongodb/lib/server.js:270:14)
    at g (events.js:260:16)
    at emitOne (events.js:77:13)
    at emit (events.js:169:7)
    at null.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb-core/lib/topologies/server.js:322:12)
    at emitOne (events.js:77:13)
    at emit (events.js:169:7)
    at null.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb-core/lib/connection/pool.js:271:12)
    at g (events.js:260:16)
    at emitTwo (events.js:87:13)
    at emit (events.js:172:7)
    at Socket.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb-core/lib/connection/connection.js:175:10)
    at Socket.g (events.js:260:16)
    at emitNone (events.js:67:13)
    at Socket.emit (events.js:166:7)
    at Socket._onTimeout (net.js:332:8)
    at _runOnTimeout (timers.js:524:11)
    at _makeTimerTimeout (timers.js:515:3)
    at Timer.unrefTimeout (timers.js:584:5)
2017-01-02T07:45:39.415Z - info: 127.0.0.1 - - [02/Jan/2017:07:45:39 +0000] "POST /api/vendortouserchat HTTP/1.1" 404 34 "-" "PostmanRuntime/3.0.5"

2017-01-02T07:46:03.923Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 13:16:03 GMT+0530 (IST), pid=17668, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=138158080, heapTotal=104991840, heapUsed=85993584, loadavg=[1.513671875, 1.77880859375, 1.95947265625], uptime=563402
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:622:28
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T07:53:39.323Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 13:23:39 GMT+0530 (IST), pid=17897, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=138227712, heapTotal=104991840, heapUsed=85340160, loadavg=[2.09521484375, 1.92626953125, 1.9462890625], uptime=563858
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:622:28
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T07:55:38.974Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 13:25:38 GMT+0530 (IST), pid=17931, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=139169792, heapTotal=104991840, heapUsed=85286512, loadavg=[2.43017578125, 2.08984375, 2.00439453125], uptime=563977
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:612:38
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T07:55:58.189Z - error: uncaughtException: missing ) after argument list date=Mon Jan 02 2017 13:25:58 GMT+0530 (IST), pid=17947, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=126734336, heapTotal=95728480, heapUsed=70136928, loadavg=[2.47265625, 2.1162109375, 2.01513671875], uptime=563997
SyntaxError: missing ) after argument list
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/routes/vendorapp.server.routes.js:8:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at /Users/anshulsharma/Desktop/chibleetestbackend/config/lib/express.js:275:9
    at Array.forEach (native)
    at Object.module.exports.initModulesServerRoutes (/Users/anshulsharma/Desktop/chibleetestbackend/config/lib/express.js:274:32)
    at Object.module.exports.init (/Users/anshulsharma/Desktop/chibleetestbackend/config/lib/express.js:340:10)
    at /Users/anshulsharma/Desktop/chibleetestbackend/config/lib/app.js:46:27
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/mongo_client.js:210:11
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickCallback (node.js:349:13)
2017-01-02T07:56:35.391Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 13:26:35 GMT+0530 (IST), pid=17955, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=139227136, heapTotal=106023776, heapUsed=81684696, loadavg=[2.2978515625, 2.103515625, 2.01220703125], uptime=564034
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:622:28
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T07:58:35.389Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 13:28:35 GMT+0530 (IST), pid=17968, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=138543104, heapTotal=104991840, heapUsed=85247672, loadavg=[2.01611328125, 2.0078125, 1.98095703125], uptime=564154
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:622:28
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T08:04:17.641Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 13:34:17 GMT+0530 (IST), pid=18045, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=140873728, heapTotal=104991840, heapUsed=85745288, loadavg=[1.97412109375, 2.0126953125, 1.99365234375], uptime=564496
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:623:28
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T08:05:51.252Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 13:35:51 GMT+0530 (IST), pid=18095, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=137981952, heapTotal=104991840, heapUsed=85354224, loadavg=[1.85791015625, 1.9833984375, 1.984375], uptime=564590
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:623:28
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T08:06:53.465Z - error: uncaughtException: Cannot read property 'toString' of undefined date=Mon Jan 02 2017 13:36:53 GMT+0530 (IST), pid=18139, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=139223040, heapTotal=104991840, heapUsed=85284392, loadavg=[1.63720703125, 1.90625, 1.955078125], uptime=564652
TypeError: Cannot read property 'toString' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:623:32
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T08:07:35.547Z - error: uncaughtException: Cannot read property '_id' of undefined date=Mon Jan 02 2017 13:37:35 GMT+0530 (IST), pid=18170, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=139268096, heapTotal=104991840, heapUsed=85287104, loadavg=[1.73876953125, 1.8876953125, 1.943359375], uptime=564694
TypeError: Cannot read property '_id' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:623:31
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T08:08:13.569Z - error: uncaughtException: Cannot read property '_id' of undefined date=Mon Jan 02 2017 13:38:13 GMT+0530 (IST), pid=18188, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=138780672, heapTotal=104991840, heapUsed=85202800, loadavg=[1.69775390625, 1.86572265625, 1.931640625], uptime=564732
TypeError: Cannot read property '_id' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:623:31
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T08:08:59.706Z - error: uncaughtException: Cannot read property '_id' of undefined date=Mon Jan 02 2017 13:38:59 GMT+0530 (IST), pid=18203, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=138608640, heapTotal=104991840, heapUsed=85262416, loadavg=[2.3720703125, 2.02099609375, 1.984375], uptime=564778
TypeError: Cannot read property '_id' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:623:31
    at Query.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/model.js:3369:16)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:259:21
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/kareem/index.js:127:16
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T08:33:04.108Z - error: uncaughtException: Cannot read property '_id' of undefined date=Mon Jan 02 2017 14:03:04 GMT+0530 (IST), pid=18238, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=138280960, heapTotal=104991840, heapUsed=85326000, loadavg=[2.3095703125, 2.08544921875, 2.0107421875], uptime=566223
TypeError: Cannot read property '_id' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:625:31
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongoose/lib/query.js:2313:18
    at newTickHandler (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mpromise/lib/promise.js:234:18)
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickDomainCallback (node.js:390:13)
2017-01-02T08:34:28.667Z - error: uncaughtException: db is not defined date=Mon Jan 02 2017 14:04:28 GMT+0530 (IST), pid=18270, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=138223616, heapTotal=104991840, heapUsed=85059688, loadavg=[2.29296875, 2.12353515625, 2.02685546875], uptime=566307
ReferenceError: db is not defined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:610:11
    at IncomingMessage.respond (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/fcm-node/lib/fcm.js:103:25)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:921:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickDomainCallback (node.js:397:17)
2017-01-02T08:35:10.609Z - error: uncaughtException: Unexpected token var date=Mon Jan 02 2017 14:05:10 GMT+0530 (IST), pid=18287, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=126095360, heapTotal=95728480, heapUsed=70161032, loadavg=[2.626953125, 2.20458984375, 2.0576171875], uptime=566349
SyntaxError: Unexpected token var
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/routes/vendorapp.server.routes.js:8:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at /Users/anshulsharma/Desktop/chibleetestbackend/config/lib/express.js:275:9
    at Array.forEach (native)
    at Object.module.exports.initModulesServerRoutes (/Users/anshulsharma/Desktop/chibleetestbackend/config/lib/express.js:274:32)
    at Object.module.exports.init (/Users/anshulsharma/Desktop/chibleetestbackend/config/lib/express.js:340:10)
    at /Users/anshulsharma/Desktop/chibleetestbackend/config/lib/app.js:46:27
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/mongo_client.js:210:11
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickCallback (node.js:349:13)
2017-01-02T08:35:19.206Z - error: uncaughtException: db.collection(...).find(...).exec is not a function date=Mon Jan 02 2017 14:05:19 GMT+0530 (IST), pid=18290, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=139264000, heapTotal=104991840, heapUsed=85108992, loadavg=[2.57666015625, 2.201171875, 2.05712890625], uptime=566358
TypeError: db.collection(...).find(...).exec is not a function
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:614:14
    at IncomingMessage.respond (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/fcm-node/lib/fcm.js:103:25)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:921:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickDomainCallback (node.js:397:17)
2017-01-02T08:36:02.043Z - error: uncaughtException: Cannot read property '_id' of undefined date=Mon Jan 02 2017 14:06:02 GMT+0530 (IST), pid=18303, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=141938688, heapTotal=107055712, heapUsed=75076072, loadavg=[2.3662109375, 2.2138671875, 2.06982421875], uptime=566401
TypeError: Cannot read property '_id' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:627:31
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/utils.js:96:12)
    at Collection.find (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/collection.js:355:44)
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:612:41
    at IncomingMessage.respond (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/fcm-node/lib/fcm.js:103:25)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:921:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickDomainCallback (node.js:397:17)
2017-01-02T08:38:14.568Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 14:08:14 GMT+0530 (IST), pid=18347, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=137814016, heapTotal=104991840, heapUsed=85318352, loadavg=[2.4697265625, 2.228515625, 2.09228515625], uptime=566533
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:625:28
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/utils.js:96:12)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/collection.js:1364:5
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/utils.js:96:12)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/cursor.js:662:5
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:156:5)
    at setCursorNotified (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:506:3)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:572:16
    at Object.queryCallback [as cb] (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:238:5)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:443:18
    at authenticateStragglers (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:399:16)
    at null.messageHandler (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:421:5)
    at Socket.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/connection.js:294:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:536:20)
2017-01-02T08:39:35.872Z - error: uncaughtException: Cannot read property '_id' of null date=Mon Jan 02 2017 14:09:35 GMT+0530 (IST), pid=18367, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=138534912, heapTotal=106023776, heapUsed=81680320, loadavg=[1.99609375, 2.1640625, 2.0771484375], uptime=566614
TypeError: Cannot read property '_id' of null
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:626:28
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/utils.js:96:12)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/collection.js:1364:5
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/utils.js:96:12)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/cursor.js:662:5
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:156:5)
    at setCursorNotified (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:506:3)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:572:16
    at Object.queryCallback [as cb] (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/cursor.js:238:5)
    at /Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:443:18
    at authenticateStragglers (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:399:16)
    at null.messageHandler (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/pool.js:421:5)
    at Socket.<anonymous> (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/node_modules/mongodb-core/lib/connection/connection.js:294:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at TCP.onread (net.js:536:20)
2017-01-02T08:39:53.299Z - error: uncaughtException: Cannot read property 'toString' of undefined date=Mon Jan 02 2017 14:09:53 GMT+0530 (IST), pid=18375, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=139632640, heapTotal=104991840, heapUsed=86412448, loadavg=[1.8544921875, 2.123046875, 2.06396484375], uptime=566632
TypeError: Cannot read property 'toString' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:626:32
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/utils.js:96:12)
    at Collection.find (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/collection.js:355:44)
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:611:41
    at IncomingMessage.respond (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/fcm-node/lib/fcm.js:103:25)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:921:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickDomainCallback (node.js:397:17)
2017-01-02T08:42:05.094Z - error: uncaughtException: Cannot read property '_id' of undefined date=Mon Jan 02 2017 14:12:05 GMT+0530 (IST), pid=18404, uid=501, gid=20, cwd=/Users/anshulsharma/Desktop/chibleetestbackend, execPath=/usr/local/bin/node, version=v4.4.7, argv=[/usr/local/bin/node, /Users/anshulsharma/Desktop/chibleetestbackend/server.js], rss=139997184, heapTotal=106023776, heapUsed=83396392, loadavg=[1.85693359375, 2.05419921875, 2.04541015625], uptime=566764
TypeError: Cannot read property '_id' of undefined
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:627:31
    at handleCallback (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/utils.js:96:12)
    at Collection.find (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/mongodb/lib/collection.js:355:44)
    at /Users/anshulsharma/Desktop/chibleetestbackend/modules/vendorapp/server/controllers/vendorapp.server.controller.js:612:41
    at IncomingMessage.respond (/Users/anshulsharma/Desktop/chibleetestbackend/node_modules/fcm-node/lib/fcm.js:103:25)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:921:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickDomainCallback (node.js:397:17)
2017-01-02T08:44:02.696Z - info: 127.0.0.1 - - [02/Jan/2017:08:44:02 +0000] "POST /v2/vendortouserchat HTTP/1.1" - - "-" "PostmanRuntime/3.0.5"

